intel_extension_for_transformers/llm/runtime/graph/application/main_pybind.cpp

```diff
@@ -72,6 +75,29 @@ class Model {
     generate_count = 0;
   }
 
+  py::array_t<float> forward(const std::vector<std::vector<model_token>>& input_ids) {         
```

Comment:
can we return vector directly to avoid introducing numpy dependency?

Reply:
`#include <pybind11/numpy.h>`
- numpy is in pybind11, not including new dependency.
- which is also used by enabling gptq models.
https://github.com/intel/intel-extension-for-transformers/pull/611/files